src/main/resources/db/demo/10_layout.sql

```
@@ -0,0 +1,142 @@
+-- =====================================================================
+-- 10. 배치 재구성 — 모든 TEXT 를 col_span 1 로
+-- ---------------------------------------------------------------------
+-- 실측 근거: col_span 1 = 297px · 2 = 610px · 3 = 922px
+--   TEXT 는 폭이 넓을수록 한 줄이 길어져 읽기 나빠진다 → 전부 1
+--   APPROVAL / IMAGE 는 카드가 풍부하거나 썸네일이 여러 장이라 넓게 둔다
+--   행마다 col_span 합 = 3 (BLK-003)
+-- =====================================================================
+
+-- 9001 채널 발굴·조건 등재 — IMAGE(9005) 를 아래 전폭으로
+UPDATE block SET row_index=0, sort_order=0, col_span=1 WHERE block_id=9001;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)